### PR TITLE
[358] Submission preview form

### PR DESCRIPTION
**Pull request by stephenhillier**

### CODE REVIEW

File: app/frontend/src/submissions/components/SubmissionForm/MethodOfDrilling.vue [hunk 1]
```diff
@@ -57,8 +57,8 @@
             <b-form-radio-group v-model="wellOrientationInput"
                                 stacked
                                 name="wellOrientationRadio">
-              <b-form-radio value="VERTICAL">Vertical</b-form-radio>
-              <b-form-radio value="HORIZONTAL">Horizontal</b-form-radio>
+              <b-form-radio value="True">Vertical</b-form-radio>
```
Comment (Sybrand) [inline app/frontend/src/submissions/components/SubmissionForm/MethodOfDrilling.vue]: I though we had decided that using the codes makes more sense than using the boolean values?

File: app/frontend/src/submissions/components/SubmissionPreview/PreviewMap.vue [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<template>
```
Comment (Sybrand) [inline app/frontend/src/submissions/components/SubmissionPreview/PreviewMap.vue]: You're going to need to pull in the esri map layers to be consistent with maps on submission and well detail.

File: app/frontend/package.json [hunk 3]
```diff
@@ -28,6 +28,7 @@
     "vue-router": "^3.0.1",
     "vue-select": "^2.5.0",
     "vue-smoothscroll": "^0.1.1",
+    "vue2-leaflet": "^1.0.2",
     "vuex": "^3.0.1"
```
Comment (Sybrand) [inline app/frontend/package.json]: You're also going to need to pull in the esri leaflet package to load in the esri layers.

File: app/frontend/src/submissions/main.js [hunk 4]
```diff
@@ -25,6 +25,8 @@ import FormInput from '@/common/components/FormInput.vue'
 import authenticate from '@/common/authenticate.js'
 import {FETCH_CONFIG} from '@/common/store/config.js'
 
+import 'leaflet/dist/leaflet.css'
```
Comment (Sybrand) [inline app/frontend/src/submissions/main.js]: Do we have to do this on main? Can it not be done on the component in question?

File: app/frontend/src/submissions/components/SubmissionPreview/SubmissionPreview.vue [hunk 5]
```diff
@@ -0,0 +1,485 @@
...
+
+        // find the subclass to get its description
+        subclass = subclassCodes.find((code) => {
+          return code.well_subclass_guid === this.form.well_subclass
+        })['description']
+      }
+
+      return subclass
+    },
+    UTM () {
```
Comment (Sybrand) [inline app/frontend/src/submissions/components/SubmissionPreview/SubmissionPreview.vue]: I think we should move this code into a common area - it's an almost exact copy of the same code in Coords.vue

**Comment by stephenhillier** [comment]
[EPIC#Submission | Change submission prior to submitting |2| !!!](https://trello.com/c/eqC2twac/206-epicsubmission-change-submission-prior-to-submitting-2)

**Pull request merged**